docs/yul.rst

```diff
@@ -935,6 +935,8 @@ the ``dup`` and ``swap`` instructions as well as ``jump`` instructions, labels a
 +-------------------------+-----+---+-----------------------------------------------------------------+
 | difficulty()            |     | F | difficulty of the current block                                 |
 +-------------------------+-----+---+-----------------------------------------------------------------+
+| prevrandao()            |     | P | randomness provided by the beacon chain                         |
```

Comment:
I think this documentation piece makes it look like that both opcodes exists and return a different value.

Reply:
Hm what would be a good way to present it here, with the one letter column allowing very limited space